clang-tools-extra/clang-tidy/readability/MagicNumbersCheck.h

```diff
@@ -78,10 +78,6 @@ class MagicNumbersCheck : public ClangTidyCheck {
         CharSourceRange::getTokenRange(MatchedLiteral->getSourceRange()),
         *Result.SourceManager, getLangOpts());
 
-    const auto parents = Result.Context->getParents(*MatchedLiteral);
-
-    const auto val = parents[0].template get<CallExpr>();
-
```

Comment:
Was this deadcode? seems unrelated to this change

Reply:
Yep, it's deadcode added in one of earlier revisions